lib/elixir/lib/exception.ex

```diff
@@ -1997,8 +1997,22 @@ defmodule Protocol.UndefinedError do
 
   @impl true
   def message(%{protocol: protocol, value: value, description: description}) do
-    "protocol #{inspect(protocol)} not implemented for #{inspect(value)} of type " <>
-      value_type(value) <> maybe_description(description) <> maybe_available(protocol)
+    inspected =
+      value
+      |> inspect(pretty: true)
+      |> String.replace(~r/^(?=.+)/m, "    ")
```

Comment:
If there's a simpler way of doing this, please let me know. Basically this does a lookahead to only indent lines that have content on them. Lines without content won't get spaces added.

This is mostly to make the formatter happy :)

Reply:
`|> String.replace("\n", "\n    ")` would probably do it too.

Reply:
```suggestion
      # Indent only lines with contents on them
      |> String.replace(~r/^(?=.+)/m, "    ")
```

Reply:
Generally I do `|> String.split(["\r\n"])` and then see if it is empty or not but a Regex also works.